# WGCNA Application to Proteomics and Metabolomics Data
Progresses in mass spectrometric instrumentation and bioinformatics identification
algorithms made over the past decades allow quantitative measurements of relative
or absolute protein/metabolite amounts in cells in a high-throughput manner, which
has significantly expedited the exploration into functions and dynamics of complex biological
systems. However, interpretation of high-throughput data is often restricted by
the limited availability of suitable computational methods and enough statistical power.
While many computational methodologies have been developed in the past decades
to address the issue, it becomes clear that network-focused rather than individual gene/
protein-focused strategies would be more appropriate to obtain a complete picture of
cellular responses. Recently, an R analytical package named as weighted gene coexpression
network analysis (WGCNA) was developed and applied to high-throughput microarray
or RNA-seq datasets since it provides a systems-level insights, high sensitivity to
low abundance, or small fold changes genes without any information loss. The
approach was also recently applied to proteomic and metabolomic data analysis. However,
due to the fact that low coverage of the current proteomic and metabolomic analytical
technologies, causing the format of datasets are often incomplete, the method
needs to be modified so that it can be properly utilized for meaningful biologically interpretation.
In this chapter, we provide a detailed introduction of the modified protocol
and its tutorials for applying the WGCNA approach in analyzing proteomic and
metabolomic datasets.

## Citation
Pei G., Chen L., Zhang W.. WGCNA Application to Proteomic and Metabolomic Data Analysis. Methods Enzymol. 2017, 585:135-158. https://pubmed.ncbi.nlm.nih.gov/28109426/
